package-lock.json

```diff
@@ -621,6 +646,12 @@
         "ms": "2.0.0"
       }
     },
+    "decamelize": {
```

Comment:
What are these new packages used for? Not obvious to me from code changes (but maybe it should be).

Reply:
this one is a transient dependency. The new packages are used in mocha.opts to enable running TS tests in mocha